src/style.css

```diff
@@ -1 +1,162 @@
-/* empty source */
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: fit-content;
+  padding: 16px;
+
+  background: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  height: 134px;
+  width: 160px;
+  margin: 16px auto 0;
+}
+
+.card__text {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  margin: 0;
+}
+
+.card__text--1 {
+  width: 166px;
+  height: 36px;
+  line-height: 18px;
+  margin-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+}
+
+.card__text--2 {
+  height: 14px;
+  margin-top: 4px;
+  font-weight: 400;
+  line-height: 14px;
+  font-size: 10px;
+  color: #616070;
+}
+
+.card__rate {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-top: 16px;
+  width: 166px;
+}
+
+.stars {
+  display: flex;
+  line-height: 14px;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(./images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin: 0 2px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__rate-review {
+  width: 53px;
+  height: 16px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+
+  display: flex;
+  text-align: right;
+  justify-content: flex-end ;
+  flex: none;
+  order: 1;
+  flex-grow: 0;

```

Comment:
it looks like all these are redundant - you could simply use a `<span` with text-align
but that's a thing to consider - no block for an approve